module/VuFind/src/VuFind/Db/Entity/SearchEntityInterface.php

```diff
@@ -40,4 +42,170 @@
  */
 interface SearchEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get user.
+     *
+     * @return int
+     */
+    public function getUserId(): int;
```

Comment:
For consistency with other interfaces, this should be getUser() returning a UserEntityInterface (with equivalent changes to the setter below).